conf/script/ha_stop_io

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+HA_LOG="/var/log/seagate/cortx/ha"
+
+# Extract keys matching to STR from consul that contain the string stop_io
+STR="cortx/base/ha/obj/node/"
+# search in consul for "action": "stop_io"
+OUTPUT=$( consul kv get -recurse $STR | grep "\"action\": \"stop_io\"" )
```

Comment:
It will be helpful to log output of "consul kv get -recurse $STR" in debugging.
I guess, the logging code needs to be moved up and then log the output of above command.

Reply:
Done